src/coreclr/src/jit/lsra.cpp

```diff
@@ -4435,7 +4498,24 @@ void LinearScan::spillGCRefs(RefPosition* killRefPosition)
         {
             continue;
         }
-        unassignPhysReg(regRecord, assignedInterval->recentRefPosition);
+        bool needsKill = varTypeIsGC(assignedInterval->registerType);
```

Comment:
I think that was part was already merged in #679, why does github show that as a diff?

Reply:
I believe it's because this is actually showing the original change that I extracted for that PR. I haven't yet pushed a rebased version of this PR.